lib/still/compiler/compilation_stage.ex

```diff
@@ -1,32 +1,53 @@
 defmodule Still.Compiler.CompilationStage do
   @moduledoc """
+  Every compilation request goes through `CompilationStage`. This process is
```

Comment:
This is no longer true. Some compilation requests do not go through CompilationStage. We need to find a better way to do it, but on the view helpers, there are some direct calls to compile other files.

Reply:
Updated it. Each of these cases is documented in the relevant module.